chef/cookbooks/cpe_profiles/resources/cpe_profiles.rb

```diff
@@ -61,7 +61,14 @@ def find_managed_profile_identifiers
     end
   end
   current_identifiers = []
-  profiles_string = shell_out!('profiles -P -o stdout-xml')
+  if node.os_at_least?('10.13')
+    profiles_string = shell_out!('profiles list -output stdout-xml')
+  elsif node.os_at_least?('10.10')
+    profiles_string = shell_out!('profiles -P -o stdout-xml')
+  else
+    profiles_string = shell_out!('profiles -P -o stdout-xml; '\
+      'cat stdout-xml.plist')
```

Comment:
It would be better to manage the location this file is being created at (put it into the chef cache location) and then reading the file from there. I know it is more work, but having stdout-xml.plist at a random location on disk isn't ideal.